### PR TITLE
Add enhancements to Python plotting scripts to release branch

### PR DESCRIPTION
**Pull request by BenjaminBlake-NOAA**
## DESCRIPTION OF CHANGES: 
Output files generated by plot_allvars and plot_allvars_diff are now written out to the expt_dir rather than the Python directory under regional_workflow/ush/Python. In addition, logic has been added to generate plots for a defined set of forecast hours. The user must now specify the starting/ending forecast hours as well as the forecast hour increment.  These changes have been merged into the develop branch, but also need to be added to the release branch.

## TESTS CONDUCTED: 
These changes were tested on both Hera and Jet.

## ISSUE (optional): 
Resolves issue [376](https://github.com/NOAA-EMC/regional_workflow/issues/376)

## CONTRIBUTORS (optional): 
David Wright, Jamie Wolff

### CODE REVIEW

**Comment by BenjaminBlake-NOAA** [comment]
@jwolff-ncar Do we also want to include the batch submission scripts from PR [412](https://github.com/NOAA-EMC/regional_workflow/pull/412) in this PR?  If so, I could add them to this one after that gets merged.

**Comment by jwolff-ncar** [comment]
That's a great idea! I have a new version of those scripts ready for review so let me check those in and once they are merged it would be great to have you combine them with this PR. Thanks!

**Comment by BenjaminBlake-NOAA** [comment]
@jwolff-ncar Ok great!  Once you add those I will test them on Hera.  I just finished testing the current versions and they worked well for me, though I think the versions of the Python scripts in your feature/python_subscripts branch were older versions.

**Comment by BenjaminBlake-NOAA** [comment]
@jwolff-ncar  I just added the batch submission scripts to this PR.

File: ush/Python/plot_allvars.py [hunk 1]
```diff
@@ -9,13 +9,15 @@
 #           David Wright 	Org: University of Michigan
 #
 # Instructions:		Make sure all the necessary modules can be imported.
-#                       Four command line arguments are needed:
+#                       Five command line arguments are needed:
```
Comment (jwolff-ncar) [inline ush/Python/plot_allvars.py]: I believe this should be "Six"

Comment (BenjaminBlake-NOAA) [reply ush/Python/plot_allvars.py]: Yep, thanks for catching that.  I just fixed it.

**Comment by jwolff-ncar** [comment]
I cannot approve, but I think this looks good.

**Comment by JeffBeck-NOAA** [comment]
@jwolff-ncar, can you approve now?

**Comment by jwolff-ncar** [comment]
@JeffBeck-NOAA Still do not see that option. That's ok. Someone else can do it. I will just add comments when I have time to review.

**Comment by JeffBeck-NOAA** [comment]
@jwolff-ncar, OK, you should have that option now.  I'll go ahead and approve the PR anyway.

**Review by JeffBeck-NOAA — APPROVED** [review]

**Comment by BenjaminBlake-NOAA** [comment]
Alright I will go ahead and merge this PR, thanks @jwolff-ncar and @JeffBeck-NOAA

**Pull request merged**